test/PackageGraph.js

```diff
@@ -0,0 +1,76 @@
+import log from "npmlog";
+
+// file under test
+import Package from "../src/Package";
+import PackageGraph from "../src/PackageGraph";
+
+// silence logs
+log.level = "silent";
+
+describe("PackageGraph", () => {
+
+  const createPackages = function(version, dependencyVersion) {
+    dependencyVersion = dependencyVersion || version;
+
+    return [
+      new Package(
+        {
+          name: "my-package-1",
+          version: version,
+          bin: "bin.js",
+          scripts: { "my-script": "echo 'hello world'" },
+          dependencies: { "my-dependency": "^1.0.0" },
+          devDependencies: { "my-dev-dependency": "^1.0.0" },
+          peerDependencies: { "my-peer-dependency": "^1.0.0" }
+        },
+        "/path/to/package1"
+      ),
+      new Package(
+        {
+          name: "my-package-2",
+          version: version,
+          bin: "bin.js",
+          scripts: { "my-script": "echo 'hello world'" },
+          dependencies: { "my-dependency": "^1.0.0" },
+          devDependencies: { "my-package-1": dependencyVersion },
+          peerDependencies: { "my-peer-dependency": "^1.0.0" }
+        },
+        "/path/to/package2"
+      )
+    ];
+  };
+
+  describe("constructor", () => {
+    it(".get should return dependencies", () => {
+      const [ pkg1, pkg2 ] = createPackages("0.0.1");
+      const graph = new PackageGraph([ pkg1, pkg2 ]);
+
+      expect(graph.get(pkg1.name).dependencies).toEqual([]);
+      expect(graph.get(pkg2.name).dependencies).toEqual([pkg1.name]);
+    });
+
+    it(".get should not return the dependencies for urecognized versions", () => {
+      const [pkg1, pkg2] = createPackages("0.0.1", "github:user-foo/project-foo#v0.0.1");
```

Comment:
To keep consistency, add spaces before and after `[]` like in the test before.

Reply:
Also, I am not sure if you need this test if you have the second `.get should not return the dependencies for urecognized versions` more detailed test.